tests/Metadata/TargetsMetadataTest.php

```diff
@@ -48,15 +49,15 @@ public function testGetHashesAndLength(): void
         try {
             $metadata->getHashes('void.txt');
             $this->fail('Exception was not thrown for an invalid target.');
-        } catch (\InvalidArgumentException $e) {
-            $this->assertSame("Unknown target: 'void.txt'", $e->getMessage());
+        } catch (NotFoundException $e) {
+            $this->assertSame("Target not found: void.txt", $e->getMessage());
         }
 
         try {
             $metadata->getLength('void.txt');
             $this->fail('Exception was not thrown for an invalid target.');
-        } catch (\InvalidArgumentException $e) {
-            $this->assertSame("Unknown target: 'void.txt'", $e->getMessage());
+        } catch (NotFoundException $e) {
+            $this->assertSame("Target not found: void.txt", $e->getMessage());
```

Comment:
Small nit is removing the single quote around the file name intentional?

Reply:
It is not. NotFoundException has its own logic to determine how the exception message is formatted, and it doesn't put the single quotes in.